src/rich-text-editor.component.js

```diff
@@ -27,6 +27,21 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
     }
   }
 
+  function interceptPaste (event) {
+    // https://developer.mozilla.org/en-US/docs/Web/Events/paste
+    event.preventDefault()
+    event.stopPropagation()
+    let paste = (window.clipboardData).getData('text');
```

Comment:
MDN has `e.clipboardData || window.clipboardData` but I found that e.clipboardData would not transfer HTML.